src/client/sandbox/code-instrumentation/location/wrapper.js

```diff
@@ -51,6 +53,9 @@ export default class LocationWrapper extends EventEmitter {
             return ensureTrailingSlash(resolveElement.href, locationUrl);
         };
         const getProxiedHref = href => {
+            if (domProcessor.JAVASCRIPT_PROTOCOL_REG_EX.test(href))
```

Comment:
Let's move this to the `domProcessor.isJsProtocol (value)` static method and remove `JAVASCRIPT_PROTOCOL_REG_EX` from domProcessor's constructor.